frontend/www/js/omegaup/common/runs_chart.js

```diff
@@ -0,0 +1,86 @@
+import common_RunsChart from '../components/common/RunsChart.vue';
+import { API, UI, OmegaUp, T } from '../omegaup.js';
+import Vue from 'vue';
+
+OmegaUp.on('ready', function() {
+  let commonRunsChart = new Vue({
+    el: '#runs-chart',
+    render: function(createElement) {
+      return createElement('omegaup-common-runschart', {
+        props: {
+          chartOptions: this.chartOptions,
+        },
+      });
+    },
+    data: {
+      chartOptions: {},
+    },
+    components: {
+      'omegaup-common-runschart': common_RunsChart,
+    },
+  });
+
+  API.Run.counts()
+    .then(series => {
+      if (series.total.length === 0) return;
+
+      let dataInSeries = [];
+      let acInSeries = [];
+      for (let i in series.total) {
+        if (series.total.hasOwnProperty(i)) {
+          dataInSeries.push(parseInt(series.total[i]));
+        }
+        if (series.ac.hasOwnProperty(i)) {
+          acInSeries.push(parseInt(series.ac[i]));
+        }
+      }
+
+      let minDate = new Date();
+      minDate.setDate(minDate.getDate() - 30 * 3);
+
+      let minY = dataInSeries[0] - dataInSeries[0] * 0.5;
```

Comment:
```suggestion
      let minY = dataInSeries[0] / 2.0;
```

esto es equivalente y más fácil de entender.

Reply:
Listo